VUB-HYDR/Wikimpacts
# 173 Validate categorical fields Main_Event and Hazards

This PR adds validation for the two categorical fields "Main_Event" and "Hazards".
- "Main_Event" is checked in the schema -- validation at schema level prevents any incorrect values from ending up in the database. The same could not be done at the moment for "Hazards" because it's of type `OBJECT`. 
- "Hazards" and "Main_Event" are both filtered and cleaned while parsing in `Database/parse_events.py`
- The schema of `impactdb.v1.raw.db` is **not updated to reflect the newly added validation rules** because sqlite3 does not natively support adding constraints to existing tables -- so this will need to be tested with the next database full run.
- **_NEW:_** Hazards that don't correspond to the Main_Event are filtered out.

To test this PR: 
- Try parsing an existing scripts. It's useful to manually add some incorrect examples and inspect the results
- ~~Try inserting into the table and see if errors are thrown~~
